### PR TITLE
new IOJones cask

### PR DESCRIPTION
**Pull request by ghost**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/iojones.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'iojones' do
+  version '1.1'
+  sha256 '381cd0f643c89f016e30cf978d361dcf06c396d6d1ed76e9ae9c192a3ed060b7'
+
+  url "https://jaist.dl.sourceforge.net/project/iojones/IOJones/1.1/IOJones.zip"
```
Comment (vitorgalvao) [inline Casks/iojones.rb]:
```suggestion
  url "https://jaist.dl.sourceforge.net/project/iojones/IOJones/#{version}/IOJones.zip"
```

This is why you need to follow the template. Doing so would have caught this error.

Comment (vitorgalvao) [reply Casks/iojones.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/iojones.rb (hunk 1 above)
Comment (mrjayviper) [reply Casks/iojones.rb]: added

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
> SourceForge URL format incorrect. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls

Again, you need to run the commands in the template. It exists to be followed, not to be ignored. Following it saves everyone’s time.

**Comment by vitorgalvao** [comment]
Closing for lack of reply.

**Pull request closed without merging**